CMakeLists.txt

```diff
@@ -36,8 +41,16 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "RelWithDebInfo" CACHE STRING "Choose the type of build, options are: Debug RelWithDebInfo." FORCE)
 endif()
 set(VERSION_TAG "" CACHE STRING "Build date for nightly versioning")
-set(EXTRA_DATA_DIR "" CACHE STRING "Directory for externalized data")
-set(INSTALL_LIB_DIR lib CACHE PATH "Installation directory for libraries")
+if (UNIX AND NOT MINGW AND NOT APPLE)
+	set(EXTRA_DATA_DIR "${CMAKE_INSTALL_FULL_DATADIR}/ja2" CACHE STRING "Directory for externalized data (usually just set CMAKE_INSTALL_DATADIR instead)")
+else()
+	set(EXTRA_DATA_DIR "" CACHE STRING "Directory for externalized data")
+endif()
+if (UNIX AND NOT MINGW AND NOT APPLE)
+	set(INSTALL_LIB_DIR ${CMAKE_INSTALL_FULL_LIBDIR})
```

Comment:
why not cached?

Reply:
Because it's bad to be able to set both variables. That may accidentially lead to a broken setup. It's best to hide the internals as much as possible.

Reply:
It's inconsistent, please make it always or never cached.

Reply:
I'm fine with making it never cached, but I don't know the impact on non-linux systems.